### PR TITLE
Feature sections: layout & content udpate

### PR DESCRIPTION
**Pull request by buressjan**

### CODE REVIEW

File: src/sections/organisers/Organisers.content.tsx
Comment (SedlarDavid) [inline src/sections/organisers/Organisers.content.tsx:12]: SU UTB

**Review by SedlarDavid — CHANGES_REQUESTED** [review]
Jen to SU UTB, ale nejsem frontendakt, pockej na Fifiho feedback

File: src/sections/contact/Contact.styled.tsx
Comment (tomesfilip) [inline src/sections/contact/Contact.styled.tsx:52]: is there a need for `!important` ?

**Review by tomesfilip — CHANGES_REQUESTED** [review]
1. UTB logo - please use .svg format
2. .firebase cache should be in .gitignore IMO
3. logos could also be in .svg format - at least the `flyUnited` (https://www.flyunited.cz/wp-content/themes/flyunited/Assets/img/logotype.svg) 
4. why there are logos with .pdf and .ai extensions? these are not supported by browsers
5.

File: src/sections/contact/Contact.styled.tsx
Comment (buressjan) [reply src/sections/contact/Contact.styled.tsx:55]: Yes, but it is true, I can also switch condition with color prop below

File: src/sections/program/Program.content.tsx
Comment (buressjan) [reply src/sections/program/Program.content.tsx:60]: Strangely formatted here, I got it on single line in vscode

File: src/App.css
Comment (SedlarDavid) [inline src/App.css:69]: As @tomesfilip  pointed earlier, is important! necesarry?

File: src/App.tsx
Comment (SedlarDavid) [inline src/App.tsx:26]: Maybe better naming, Consent is general and might be used for other actions as well

File: src/components/consent/Consent.tsx
Comment (SedlarDavid) [inline src/components/consent/Consent.tsx:11]: This naming is just fine, maybe update just the wrapper

File: src/components/layout/navbar/Navbar.content.tsx
Comment (SedlarDavid) [inline src/components/layout/navbar/Navbar.content.tsx:19]: You could do this with just some .prepend() function and add #

File: yarn.lock
Comment (SedlarDavid) [inline yarn.lock:1239]: will we use yarn i future or just npm since build is connected to npm also?
@tomesfilip

**Review by SedlarDavid — APPROVED** [review]
Just minor things, but wait for @tomesfilip  review

**Comment by SedlarDavid** [comment]
@buressjan I added [5b5d911](https://github.com/SU-UTB/web-ples-2023/pull/8/commits/5b5d9114d63366fe3d78c4f1e4ff86b4aadad4f0) to resolve merge conflict with origin/devel, so you can merge right away after review

File: src/functions/revealImg.tsx
Comment (tomesfilip) [inline src/functions/revealImg.tsx:2]: use const for all variables (except i) as there are no value changes done afterward

File: src/functions/revealLeft.tsx
Comment (tomesfilip) [inline src/functions/revealLeft.tsx:2]: use const for all variables (except i) as there are no value changes done afterward

File: src/sections/partners/Partners.content.tsx
Comment (tomesfilip) [inline src/sections/partners/Partners.content.tsx:18]: for now OK, but normally it's better to approach to save the logo locally and serve it like that instead of a link (if there is a small change in the URL in the future, the image will be broken)

File: src/App.css
Comment (buressjan) [reply src/App.css:69]: Here not, sorry...

File: src/components/layout/navbar/Navbar.content.tsx
Comment (buressjan) [reply src/components/layout/navbar/Navbar.content.tsx:19]: I know, but there are special characters, lowercase, and # ... so 2-3 changes, the new prop is better here IMO

File: yarn.lock
Comment (tomesfilip) [reply yarn.lock:1239]: we can use yarn. It's just package manager tool just like npm, it will run the build command that executes another command. These are defined in package.json under "scripts"

```
"scripts": {
    "dev": "vite",
    "build": "tsc && vite build",
    "preview": "vite preview"
  },
```

**Review by tomesfilip — CHANGES_REQUESTED** [review]
wouldnt' be better to have one reveal function and pass needed arguments to it? 

```
const revealFrom = (selector: string, elementVisible: number, ...) => {
const reveals = document.querySelectorAll(selector);

  for (let i = 0; i < reveals.length; i++) {
    const windowHeight = window.innerHeight;
    const elementTop = reveals[i].getBoundingClientRect().top;
    
    if (elementTop < windowHeight - elementVisible) {
      reveals[i].classList.add('done');
    } else {
      reveals[i].classList.remove('done');
    }
  }
} 
```

**Review by tomesfilip — APPROVED** [review]
GJ

**Pull request merged**
